app/events/event/route.js

```diff
@@ -1,10 +1,19 @@
 import Route from '@ember/routing/route';
+import { get } from '@ember/object';
+import { reads } from '@ember/object/computed';
+import { inject as service } from '@ember/service';
 
 export default Route.extend({
+  fastboot: service(),
+  isFastBoot: reads('fastboot.isFastBoot'),
+
   titleToken(model) {
-    return model.get('title');
+    return get(model, 'title');
```

Comment:
Out of curiosity, what's the benefit to this change?

Reply:
i think it changed from object to an ember object, which made the change necessary.